kubernetes/test-infra/kops-on-gce/kops-create.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+export NODE_SIZE=n1-standard-1
+export MASTER_SIZE=n1-standard-2
+export KOPS_STATE_STORE="gs://${BUCKET_NAME}"
+export KOPS_FEATURE_FLAGS=AlphaAllowGCE
+
+# Auth for kops
+echo $GOOGLE_CREDENTIALS > $TMP_CREDS_PATH
+export GOOGLE_APPLICATION_CREDENTIALS=$TMP_CREDS_PATH
+
+# Auth for gsutil/gcloud
+gcloud auth activate-service-account $(echo $GOOGLE_CREDENTIALS | jq -r .client_email) --key-file=$TMP_CREDS_PATH
+gcloud config set pass_credentials_to_gsutil true
```

Comment:
This scares me only a little, because it changes the host's gcloud configuration without changing it back, as far as I can tell, but I have no better suggestion. `gsutil` doesn't leave us many options.

Reply:
I hear what you're saying. It's sad that we can't just simply pass an ENV variable with JSON and isolate the environment. I think that disabling `pass_credentials_to_gsutil` (at the end of this script) would probably cause more troubles at this point as multiple kops jobs may happen to run at the same time.

I'm open to ideas though.

Reply:
I don't know that I have ideas, and honestly think this is as good as we can do given the tooling at hand. But wanted to call it out for future code archaeology to show a decision was made. :)